tasks/functions/functions.js

```diff
@@ -100,27 +152,39 @@ function discountFunction(percentage) {
  */
 
 const myObject = {
-	name: 'John',
-	lastName: 'Doe',
-	age: 25,
-	friends: ['Mike', 'Alan', 'Daniel'],
-	keys() {
-		//write your code here
-	},
-	call() {
-		//write your code here
-	}
-
+  name: "John",
+  lastName: "Doe",
+  age: 25,
+  friends: ["Mike", "Alan", "Daniel"],
+  keys() {
+    for (let key in myObject) {
```

Comment:
You should use `this` here instead of `myObject` because if you need to rename variable `myObject` you will be independent for renaming inner variable for enumeration.